modules/plan.nix

```diff
@@ -123,10 +128,6 @@ let
       type = nullOr string;
       default = (def.postHaddock or null);
     };
-    shellHook = mkOption {
```

Comment:
Do we really want to remove this? Is setting a `shellHook` on the package level pointless?

Reply:
Ok. I'm stupid. Ignore this comment :-)